test/unit/gateways/element_test.rb

```diff
@@ -10,7 +10,8 @@ def setup
     @options = {
       order_id: '1',
       billing_address: address,
-      description: 'Store Purchase'
+      description: 'Store Purchase',
+      card_present_code: 'ManualKeyed'
```

Comment:
it seems Vantiv/Mercury updated their API recently to either separate `card_present_code` from `card_input_code`, or now properly validates these values. 

fyi
CardPresentCode 0/1/2/3 Default/Unknown/Present/NotPresent
CardInputCode 0/1/2/3/4/5 Default/Unknown/MagstripeRead/ContactlessMagstripeRead/ManualKeyed/ManualKeyedMagstripeFailure

I'm working on my own changes to fix Check/ACH, `store` and more. I'll try to remember to clean up my changes once they are stable, make a PR and link this